src/index.ts

```diff
@@ -69,8 +72,10 @@ export default class FocusOverlay {
       alwaysActive: false,
       // Reposition focus box on transitionEnd for focused elements
       watchTransitionEnd: true,
+      // Reposition focus box on animationEnd based on the window object
+      watchAnimationEnd: true,
       // Reposition focus box on scroll event (debounce: default 150ms)
-      debounceScroll: true,
+      debounceScroll: false,
```

Comment:
Is there a reason why you set it to false? Is it possible you will have to change the default comment above?

Reply:
No reason, I forgot to revert this change.
I will adjust it.